theano/tensor/var.py

```diff
@@ -525,7 +525,7 @@ def __getitem__(self, args):
                         new_args.append(arg)
                 view = self.dimshuffle(pattern)
                 check_rval = [arg == slice(None, None, None) for arg in new_args]
-                if all(check_rval) == True:
```

Comment:
off topic, but test_flake8.py failed:
use 
`if <bool> is True:` or `if <bool>:` instead of `if <bool> == True:`